go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/aws/aws-sdk-go v1.29.29
 	github.com/davecgh/go-spew v1.1.1
 	github.com/google/go-cmp v0.4.0
+	github.com/pkg/errors v0.9.1
```

Comment:
I think this is something to do with finalizer #495 (I am going to refactor in another PR

Reply:
I think we should run `go mod tidy` in CI and fail the build if go.mod changes to prevent things like this in the future.

Reply:
agreed